proto/metadata.proto

```diff
@@ -298,4 +298,15 @@ message Metadata {
   // The `platform` field for each `platform_exceptions` should be mutually
   // exclusive. Duplicate matches will result in a test failure.
   repeated PlatformExceptions platform_exceptions = 5;
+
+  enum Tags {
```

Comment:
https://protobuf.dev/programming-guides/proto3/#default

```
For enums, the default value is the first defined enum value, which must be 0.
```

Based on the comment below, I suppose there should be "all = 0" for empty `tag` ?